dali/python/nvidia/dali/external_source.py

```diff
@@ -577,6 +575,11 @@ def __call__(
             kwargs = {"no_copy": no_copy}
             group = _ExternalSourceGroup(callback, source_desc, True, **group_common_kwargs)
             for i in range(self._num_outputs):
+                if dtype is not None:
```

Comment:
Instead of using global spec, now I use those kwargs which are propagated to the individual specs by op_instances